package.json

```diff
@@ -75,6 +75,7 @@
     "storybook": "storybook dev -p 7008",
     "build-storybook": "npm run build:storybook-exports && storybook build",
     "deploy-storybook": "storybook-to-ghpages",
+    "chromatic:local": "chromatic -t $CHROMATIC_PROJECT_TOKEN",
```

Comment:
Why just `local`? Wouldn't it work if we use if in the action this way?

Reply:
is it their default cli command?

Reply:
cause using only `chromatic` would override it

Reply:
ohhh got it